keycloak/keycloak-admin-ui
# Initial version of the identity providers section

## Motivation

Adds the ability to create list and delete identity providers + the ability to change the display order.

## Brief Description

see [marvel](https://marvelapp.com/prototype/55c2d8f/screen/75697040)

## Verification Steps

Go to the identity provider section click on one of the providers then save to see the list create another to verify the order change dialog.

## Checklist:


## Additional Notes
